11.coreboot.u-root.systemboot/README.md

```diff
@@ -43,7 +43,39 @@ For example, you may want to include static builds of `kexec` or `flashrom`, tha
 
 ## Building a suitable Linux kernel
 
-TODO
+TODO add sample `.config`.
+
+You need a relatively recent kernel. Ideally a kernel 4.16, to have support for VPD variables, but a 4.11 can do the job too, if you don't care about boot entries and want "brute-force" booting only.
+
+We will build a kernel with the following properties:
+
+* small enough to fit most flash chips
+* that can run Go programs (mainly futex and epoll support)
+* with the relevant storage and network drivers
+* with kexec support, so it can boot a new kernel
+* with kexec signature verification disabled (optional)
+* with devtmpfs enabled, since we don't use udev
+* XZ support to decompress the embedded initramfs
+* VPD (Vital Product Data) (optional)
+* TPM support (optional)
+* embed the u-root initramfs
+* and last but not least, "linuxboot" as default host name :)
+
+
+Steps:
+* create a minimal configuration, see https://tiny.wiki.kernel.org
+* enable `CONFIG_FUTEX` and `CONFIG_EPOLL`, required by Go, see https://github.com/golang/go/wiki/MinimumRequirements
```

Comment:
same here. I did not find that document to be that useful. Just put it right here. Show me a known good sample .config

Reply:
same as above - it will be included in a next commit, tracked in the TODO on top